src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -666,6 +668,36 @@ public static void setDistanceModelUpdateUrl(String url) {
         distanceModelUpdateUrl = url;
     }
 
+    /**
+     * Default class for rssi filter/calculation implementation
+     */
+    protected static Class rssiFilterImplClass = RunningAverageRssiFilter.class;
```

Comment:
I'm not sure about this - we need to create an instance in runtime, because every beacon needs its own filter object. Can you clarify, how you would change this?